### ISSUE TITLE
Install failure on Maverick 8.0.0 beta-3

### ISSUE THREAD

**Issue by juddey** [infinitered/ignite#2163]
### Describe the bug

With beta 3 installed, on linux. run following command:
```
npx ignite-cli@next new appname
   
✅ What bundle identifier? · com.appname
/home/justin/.npm/_npx/2351a7c35afcf833/node_modules/gluegun/build/index.js:15
    throw up;
    ^

TypeError: Cannot read properties of undefined (reading 'startsWith')
    at parsePath (/home/justin/.npm/_npx/2351a7c35afcf833/node_modules/ignite-cli/build/commands/new.js:121:34)
    at /home/justin/.npm/_npx/2351a7c35afcf833/node_modules/ignite-cli/build/commands/new.js:126:27
    at step (/home/justin/.npm/_npx/2351a7c35afcf833/node_modules/ignite-cli/build/commands/new.js:44:23)
    at Object.next (/home/justin/.npm/_npx/2351a7c35afcf833/node_modules/ignite-cli/build/commands/new.js:25:53)
    at fulfilled (/home/justin/.npm/_npx/2351a7c35afcf833/node_modules/ignite-cli/build/commands/new.js:16:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Same commands work successfully on beta 2 ie:
npx ignite-cli@8.0.0-beta.2 new appname

### Ignite version

8.0.0-beta-3

### Additional info

System
  platform           linux                                                
  arch               x64                                                  
  cpu                16 cores       AMD Ryzen 7 2700 Eight-Core Processor << this is it NEED MOAR CORES
  directory          react-native   /home/justin/projects/react-native    

JavaScript (and globally-installed packages)
  node                    16.17.0      /home/justin/n/bin/node 
  npm                     8.15.0       /home/justin/n/bin/npm  
    appcenter-cli         2.9.0                                
    chromedriver          88.0.0                               
    corepack              0.12.1                               
    detox-cli             18.20.1                              
    expo-cli              4.12.1                               
    gatsby-cli            2.12.29                              
    ghost-cli             1.11.0                               
    ignite-cli            7.5.0                                
    json                  11.0.0                               
    npm                   8.15.0                               
    patch-package         6.2.0                                
    prettier-standard     16.3.0                               
    prettier              2.5.1                                
    puppeteer-pdf         1.2.0                                
    react-native-app-id   0.0.5                                
    react-native-cli      2.0.1                                
    react-native-rename   2.7.0                                
    watchman              1.0.0                                
  yarn                    1.21.1       /usr/bin/yarn           
    detox-cli             10.0.7                               
    expo-cli              3.28.0                               
    gatsby                2.13.23                              
    gatsby-cli            2.7.15                               
    react-native-cli      2.0.1                                

Ignite
  ignite-cli         7.5.0        /home/justin/n/bin/ignite                        
  ignite src         build        /home/justin/n/lib/node_modules/ignite-cli/build 

Android
  java               11.0.16      /usr/bin/java            
  android home       -            /home/justin/Android/Sdk

**Comment by juddey** [comment]
Also occurs on macos.

**Comment by joshuayoes** [comment]
I was able to recreate this using `npx ignite-cli@8.0.0-beta.3 new appname`

<img width="851" alt="Screen Shot 2022-09-15 at 10 38 34 AM" src="https://user-images.githubusercontent.com/37849890/190473037-ff20e1e8-c70e-45d6-bba6-8ad882943ac6.png">

---

`npx ignite-cli@8.0.0-beta.3 doctor`
```
System
  platform           darwin
  arch               arm64
  cpu                10 cores     Apple M1 Max
  directory          Code         /Users/joshuayoes/Code

JavaScript (and globally-installed packages)
  node               18.7.0                                    /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/bin/node
  npm                8.15.0                                    /Users/joshuayoes/.asdf/plugins/nodejs/shims/npm
    ignite-cli       7.15.0
  yarn               1.22.19                                   /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/bin/yarn
  pnpm               7.5.2                                     /Users/joshuayoes/.asdf/installs/nodejs/18.7.0/bin/pnpm
                     <no pnpm global package info available>
  expo               -                                         not installed

Ignite
  ignite-cli         8.0.0-beta.3   /Users/joshuayoes/.npm/_npx/76bdfd687c5ef7b4/node_modules/.bin/ignite
  ignite src         build          /Users/joshuayoes/.npm/_npx/76bdfd687c5ef7b4/node_modules/ignite-cli/build

Android
  java               -            not installed
  android home       -            undefined

iOS
  xcode              13.4.1
  cocoapods          1.11.3       /Users/joshuayoes/.asdf/shims/pod
```

**Comment by joshuayoes** [comment]
Seems like I may have introduced a new regression.

<img width="899" alt="Screen Shot 2022-09-15 at 10 41 18 AM" src="https://user-images.githubusercontent.com/37849890/190473509-6908b908-8ab3-4ac9-a107-ea6e9fb27138.png">

I will attempt to recreate this locally before fixing it

**Comment by joshuayoes** [comment]
This has now been resolved as of 8.0.0-beta.5

<img width="617" alt="Screen Shot 2022-09-15 at 3 49 05 PM" src="https://user-images.githubusercontent.com/37849890/190521964-d9e11fe1-dea4-4f20-bdfc-bb9e274b2157.png">

**Comment by joshuayoes** [comment]
If you are still having issues with this, please let me know and re-open the issue
